image-download

```diff
@@ -79,6 +80,20 @@ def check_curl_args(args):
 def find(name, stores, latest, quiet):
     found = []
 
+    try:
+        with open(xdg_config_home('cockpit-dev', 'image-store-preference'), 'r') as fp:
+            pattern = fp.read().strip()
+            matches = [store for store in stores if pattern in store]
```

Comment:
Tiniest style issue: This should already be outside of the `with` again, as there is no further action on the file.

Reply:
Done